docs/reference/python/katana.local.graph.rst

```diff
@@ -14,8 +14,7 @@ When working with graph projections, keep in mind that:
 
 - The resulting `Graph` projection object shares property data with the original graph. If a user runs an analytics routine such as PageRank on the projection, this will add a new node property to both the projection and the original graph.
 
-- The projection will only be functional while the original graph remains in memory. Deleting the original graph while the projected one is still in use may cause erratic behavior with the projection. Make sure to delete any projections using Python’s `del` command before doing the same with the original graph.
-
+- The projection will only be functional while the original graph remains in memory. Deleting the original graph while the projected one is still in use may cause erratic behavior with the projection. 
```

Comment:
Deleting the original graph while the view is still accessible shouldn't be possible from Python. That was the point of our discussion. I will make sure this is really true in the wrapping code. I created an issue for this: KAT-4441.